.github/workflows/build-fw.yml

```diff
@@ -50,10 +50,18 @@ jobs:
       - uses: actions/checkout@v4
         with:
           path: tt-zephyr-platforms
+          fetch-depth: 0
       - uses: ./tt-zephyr-platforms/.github/workflows/prepare-zephyr
         with:
           app-path: tt-zephyr-platforms
 
+      # Fetch tags from repositories in manifest. Needed so that
+      # git describe functions correctly when creating a build.
+      - name: Fetch Tags
+        shell: bash
+        run: |
+          west forall -c 'git fetch --unshallow || true'
+
```

Comment:
```suggestion
```

Reply:
yeah, its just a removal because the tags are already fetched

Reply:
Where? The checkout is going to fetch tags for `tt-zephyr-platforms`, but we also need to fetch tags from Zephyr to get the Zephyr version printed during boot. The zephyr setup action performs a shallow clone by default, which is why I have this step